### ISSUE TITLE
isException global var use

### ISSUE THREAD

**Issue by kedbirhan** [ovotech/domain-protect#41]
the code below is copied from the manual scan folder and it uses "isExeption" variable to check if an exception is thrown when DNS lookup is performed. But all classes(e.g [aws-cname-eb.py](https://github.com/ovotech/domain-protect/blob/dev/manual-scans/aws-cname-eb.py) declaring the field do not set it to true when an exception is thrown. Am I missing something or is this a bug?
```
 elif (result==False) and (isException==True):
       suspectedDomains.append(cname_record)
       my_print(str(i)+". "+cname_record,"INFOB")
       my_print(exception_message, "INFO")
```
also in [aws-cname-eb.py](https://github.com/ovotech/domain-protect/blob/dev/manual-scans/aws-cname-eb.py) shouldn't line 86 return false. since the domain has been successfully resolved?.

**Comment by paulschwarzenberger** [comment]
Apologies for the delayed response. 

I've tested this script against correct and vulnerable DNS records for Elastic Beanstalk instances and it functions correctly.

On the first point, you're correct, I've tested and this elif block appears to be redundant - I'll tidy this up.

On the second point, if a CNAME is set up pointing to a resource which exists, a DNS query returns a CNAME which then resolves to one or more A records. If the resource doesn't exist, it only returns a CNAME record. So we're looking for the case where we don't get an A record but we do get a CNAME record. Therefore it's correct on line 86 to return True.
